.github/ISSUE_TEMPLATE/visualizations.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New Visualization Request
+about: Outline the generation of a new visualization
+labels: enhancement,visualization
+---
+
+**Please describe the background and context for this new visualization**
+A clear and concise description of what the proposed visualization is and its connection to dash app design. Ex. This visualization is for the community page of the dashboard to give the perspective of [...]
```

Comment:
maybe change to:
E.g. "This visualization is for the community page of the dashboard to give the perspective of [...]"